R/rd-r6.R

```diff
@@ -29,7 +29,9 @@ topic_add_r6_methods <- function(rd, block, env) {
   methods <- add_default_methods(methods)
 
   nodoc <- map_int(methods$tags, length) == 0
-  r6_warning(block, "undocumented R6 method[s]: %s", methods$name[nodoc])
+  if (any(nodoc)) {
+    warn_roxy_block(block, "Undocumented R6 method[s]: {methods$name[nodoc]}")
```

Comment:
This can be `method{?s}` I believe.

Reply:
Doesn't that require the object/length to before the plural suffix?